environment_cpu.yml

```
@@ -0,0 +1,35 @@
+name: graph
+channels:
+  - pytorch
+  - pyg
+  - conda-forge
+  - defaults
+dependencies:
+  - numcodecs
+  - pandas
+  - pip
+  - pyg
+  - python=3.11
+  - pytorch
+  - pytorch-cluster
+  - pytorch-scatter
+  - pytorch-sparse
+  - pytorch-spline-conv
+  - scikit-learn
+  - scipy
+  - torchvision
+  - tqdm
+  - xarray
+  - zarr
+  - h3-py
+  - numpy
+  - torch==2.2.2+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

I think you can set pytorch to be cpuonly with this instead, might be less brittle than this other way
```suggestion
  - cpuonly
```